policyengine_canada/variables/gov/provinces/nu/tax/income/credits/nu_cost_of_living_credit.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_canada.model_api import *
+
+
+class nu_cost_of_living_credit(Variable):
+    value_type = float
+    entity = Person
+    label = "Nunavut cost of living credit"
```

Comment:
Naming based on the tax form
```suggestion
    label = "Nunavut cost of living basic credit"
```